councilmatic_core/management/commands/convert_attachment_text.py

```diff
@@ -0,0 +1,118 @@
+import os
+import sys
+import subprocess
+import logging
+import logging.config
+import sqlalchemy as sa
+import datetime
+import signal
+import os
+import requests
+import textract
+import tempfile
+
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.db.models import Max
+
+from councilmatic_core.models import BillDocument
+
+logging.config.dictConfig(settings.LOGGING)
+logging.getLogger("requests").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+DB_CONN = 'postgresql://{USER}:{PASSWORD}@{HOST}:{PORT}/{NAME}'
+
+engine = sa.create_engine(DB_CONN.format(**settings.DATABASES['default']),
+                          convert_unicode=True,
+                          server_side_cursors=True)
+
+class Command(BaseCommand):
+    help = 'Converts bill attachments into plain text'
+    
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--update_all',
+            default=False,
+            action='store_true',
+            help='Add or update plain text for all bill attachments.')
+
+    def handle(self, *args, **options):
+        self.update_all = options['update_all']
+        self.connection = engine.connect()
+        self.add_plain_text()
+
+    def get_document_url(self):
+        self.connection.execute("SET local timezone to '{}'".format(settings.TIME_ZONE))
```

Comment:
Why do this here, instead of in the transactional context below?

Reply:
Why do this at all?

Reply:
Yes - this is unnecessary: we are not adding any timestamps to the instances of BillDocument. We can safely remove it.